src/utility/copy.cpp

```diff
@@ -136,4 +165,4 @@ DuckdbCopy(PlannedStmt *pstmt, const char *query_string, struct QueryEnvironment
 	auto chunk = res->Fetch();
 	*processed = chunk->GetValue(0, 0).GetValue<uint64_t>();
 	return true;
-}
+}
```

Comment:
please install editorconfig and save this file again. We want a final newline in the file. https://editorconfig.org/

Reply:
ok